### PR TITLE
SONAR-6338 Coverage in compute

### PR DESCRIPTION
**Pull request by julienlancelot**
SONAR-6338 Coverage in compute

### CODE REVIEW

File: sonar-batch-protocol/src/main/java/org/sonar/batch/protocol/output/BatchReportReader.java [hunk 1]
```diff
@@ -37,7 +38,7 @@ public BatchReportReader(File dir) {
 
   public BatchReport.Metadata readMetadata() {
     File file = fileStructure.metadataFile();
-    if (isNotAnExistingFile(file)) {
+    if (!isFileExists(file)) {
```
Comment (teryk) [inline sonar-batch-protocol/src/main/java/org/sonar/batch/protocol/output/BatchReportReader.java]: I would go for doesFileExist(file) or isFileFound(file). WDYT ?

File: sonar-batch-protocol/src/main/protobuf/batch_report.proto [hunk 2]
```diff
@@ -210,6 +210,24 @@ message Symbols {
   }
 }
 
+message Coverage {
```
Comment (teryk) [inline sonar-batch-protocol/src/main/protobuf/batch_report.proto]: why do I have the feeling it looks like the graph tests I'm trying to move on compute stack ?

File: sonar-batch-protocol/src/main/java/org/sonar/batch/protocol/output/BatchReportReader.java (hunk 1 above)
Comment (julienlancelot) [inline sonar-batch-protocol/src/main/java/org/sonar/batch/protocol/output/BatchReportReader.java]: Why not doesFileExist.

**Comment by teryk** [comment]
lgtm-bush

File: sonar-batch-protocol/src/main/java/org/sonar/batch/protocol/output/BatchReportReader.java (hunk 1 above)
Comment (teryk) [inline sonar-batch-protocol/src/main/java/org/sonar/batch/protocol/output/BatchReportReader.java]: because it's correct english ?

**Pull request merged**